src/popup/App.vue

```diff
@@ -19,12 +19,16 @@
     >
       <Header v-if="showHeader" />
 
-      <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
-        <RouterView
-          :class="{ 'show-header': showHeader }"
-          class="main"
-        />
-      </transition>
+      <div class="app-container">
```

Comment:
I don't quite get the reason of another element added to wrap the content. The CSS for this element is only width and height set to 100%. Maybe we could remove it?